CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## [1.14.1](https://github.com/brave/brave-ios/releases/tag/v1.14.1)
+
+ - Added images to new tab page. ([#1782](https://github.com/brave/brave-ios/issues/1782))
+ - Fixed wallet details not being displayed for empty wallets. ([#1852](https://github.com/brave/brave-ios/issues/1852))
+ - Fixed top bar visiblity when settings page is opened on iPhones with iOS 13. ([#1714](https://github.com/brave/brave-ios/issues/1714))
```

Comment:
nit: missed this in the original review. Should be spelled as `visibility` rather than `visiblity`

Reply:
Fixed